### DISCUSSION TITLE
已经登陆，为什么不能下载1080p视频？

### DISCUSSION THREAD

**Discussion by gui8912** [the1812/Bilibili-Evolved#4419] [提问 / Q&A]
最高只能下载720p视频，能不能下载h265格式的1080p，并合成mp4？

**Comment by hexadecimal233** [comment]
![image](https://github.com/the1812/Bilibili-Evolved/assets/62250232/9f9cd996-623c-449f-b6c6-f51991b9b9b0)

**Comment by gui8912** [comment]
用dash下载后还要自己合成，有点麻烦

**Reply by the1812** [reply]
![Snipaste_2023-09-09_11-03-18](https://github.com/the1812/Bilibili-Evolved/assets/26504152/daba2651-481b-448a-a543-120e49a73926)

**Reply by gui8912** [reply]
收到，谢谢了
